src/components/BulkEditList/BulkEditList.js

```diff
@@ -37,8 +46,12 @@ export const BulkEditList = () => {
       id="ui-bulk-edit.meta.title.uploadedFile"
       values={{ fileName: fileUploadedName }}
     />
-    :
-    <FormattedMessage id="ui-bulk-edit.meta.title" />;
+    : fileUploadedMatchedName ?
```

Comment:
please split it into 2 different statements, because it hard to read now

Reply:
Done